scripts/covid_hosp_prod.R

```diff
@@ -1 +1,90 @@
-# TODO
+# The COVID Hospitalization Production Forecasting Pipeline.
+#
+# Ran into some issues with targets:
+#   https://github.com/ropensci/targets/discussions/666#discussioncomment-9050772
+
+source("scripts/targets-common.R")
+
+
+#' Get exclusions from a JSON file for a given date
+#'
+#' @param date A date
+#' @param exclusions_json A JSON file with exclusions in the format:
+#'
+#'    {"exclusions": {"2024-03-24": "ak,hi"}}
+get_exclusions <- function(date, exclusions_json = here::here("scripts", "geo_exclusions.json")) {
+  s <- jsonlite::read_json(exclusions_json)$exclusions[[as.character(date)]]
+  if (!is.null(s)) {
+    return(s)
+  }
+  return("")
+}
+
+forecast_generation_date <- as.character(seq.Date(as.Date("2024-01-01"), Sys.Date(), by = "1 week"))
+geo_exclusions <- Vectorize(get_exclusions)(forecast_generation_date)
+
+rlang::list2(
+  tar_target(
+    aheads,
+    command = {
+      c(1:7)
```

Comment:
```suggestion
      c(1:28)
```
This should probably be 1:28, right? Guessing this was 1:7 for proof of concept